scenario/utils/relayScrollMessage.ts

```diff
@@ -108,7 +117,17 @@ export default async function relayScrollMessage(
       console.log(
         `[${governanceDeploymentManager.network} -> ${bridgeDeploymentManager.network}] Bridged over ${amount} of WETH to user ${to}`
       );
-    } else if (target === bridgeReceiver.address) {
+    } else if (target === l2WstETHGateway.address){
```

Comment:
Is this just a generic ERC20 gateway? I don't think we need a case for every ERC20 unless they all go through different gateways.

Reply:
it's specifically for bridging WstETH. Even though on the L2 side it's basically a generic ERC20 token on the L2 side it's different